tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu

```diff
@@ -0,0 +1,98 @@
+# Expecting base image to be the image built by ./Dockerfile.e3.gpu
+ARG BASE_IMAGE=""
+
+FROM $BASE_IMAGE
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+ARG PYTHON=python3.8
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.8.2
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.12
+
+ARG SMDATAPARALLEL_BINARY=https://smdataparallel.s3.amazonaws.com/binary/tensorflow/2.6.0/cu112/2021-08-26/smdistributed_dataparallel-1.2.1-cp38-cp38-linux_x86_64.whl
+
+ARG SMMODELPARALLEL_BINARY=https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/tensorflow-2.6.0/build-artifacts/2021-08-24-22-08/smdistributed_modelparallel-1.4.0-cp38-cp38-linux_x86_64.whl
+
+RUN wget https://sourceforge.net/projects/boost/files/boost/1.73.0/boost_1_73_0.tar.gz/download -O boost_1_73_0.tar.gz \
+   && tar -xzf boost_1_73_0.tar.gz \
+   && cd boost_1_73_0 \
+   && ./bootstrap.sh \
+   && ./b2 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true \
+   && cd .. \
+   && rm -rf boost_1_73_0.tar.gz \
+   && rm -rf boost_1_73_0 \
+   && cd /usr/include/boost
+
+WORKDIR /
+
+RUN ${PIP} install --no-cache-dir -U \
+  scikit-learn==0.24.2 \
+  pandas==1.2.5 \
+  "sagemaker>=2,<3" \
+  sagemaker-experiments==0.* \
+  "sagemaker-tensorflow>=2.6,<2.7" \
```

Comment:
this needs to be updated right?

Reply:
@hsl89 - we're not releasing the SM images this go-around, but can you make sure that this is set to an appropriate range for TF 2.7 so that we don't accidentally release with an old version of sagemaker-tensorflow?